src/Badge.js

```diff
@@ -1,61 +1,61 @@
+import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { space } from 'styled-system'
+import { type, space, color } from 'styled-system'
```

Comment:
I don't think `type` is part of the styled-system API 🤔

Reply:
Yeah I don't know why i put that there 🤦